charts/konk/scripts/provision.sh

```diff
@@ -1,7 +1,9 @@
-#!/bin/bash -xe
```

Comment:
This inline config was not actually setting the bash options when run with `helm test`.